create_lmodsitepackage.py

```diff
@@ -210,6 +210,73 @@
+    local accel_stack_in_modulepath = false
+
+    -- Check if we are using an EESSI version 2023 accelerator stack by checking if the $MODULEPATH contains
+    -- a path that starts with /cvmfs/software.eessi.io and contains accel/nvidia/ccNN
+    for path in string.gmatch(modulepath, '(.-):') do
+        if string.sub(path, 1, 41) == "/cvmfs/software.eessi.io/versions/2023.06" then
+            if string.find(path, "accel/nvidia/cc%d%d") then
+                accel_stack_in_modulepath = true
+                break
+            end
+        end
+    end
+    return accel_stack_in_modulepath
+end
+
+local function eessi_removed_module_warning_startup_hook(usrCmd)
+    if usrCmd == 'load' and not os.getenv("EESSI_SKIP_REMOVED_MODULES_CHECK") then
+        local CUDA_RELOCATION_MSG = [[All CUDA installations and modules depending on CUDA have been relocated to GPU-specific stacks.
+        Please see https://www.eessi.io/docs/site_specific_config/gpu/ for more information.]]
+
+        local RELOCATED_CUDA_MODULES = {
+            ['NCCL'] = CUDA_RELOCATION_MSG,
+            ['NCCL/2.18.3-GCCcore-12.3.0-CUDA-12.1.1'] = CUDA_RELOCATION_MSG,
+            ['UCX-CUDA'] = CUDA_RELOCATION_MSG,
+            ['UCX-CUDA/1.14.1-GCCcore-12.3.0-CUDA-12.1.1'] = CUDA_RELOCATION_MSG,
+            -- we also have non-CUDA versions of OSU Micro Benchmarks, so only match the CUDA version
+            ['OSU-Micro-Benchmarks/7.2-gompi-2023a-CUDA-12.1.1'] = CUDA_RELOCATION_MSG,
+            ['UCC-CUDA'] = CUDA_RELOCATION_MSG,
+            ['UCC-CUDA/1.2.0-GCCcore-12.3.0-CUDA-12.1.1'] = CUDA_RELOCATION_MSG,
+            ['CUDA'] = CUDA_RELOCATION_MSG,
+            ['CUDA/12.1.1'] = CUDA_RELOCATION_MSG,
+            ['CUDA-Samples'] = CUDA_RELOCATION_MSG,
+            ['CUDA-Samples/12.1-GCC-12.3.0-CUDA-12.1.1'] = CUDA_RELOCATION_MSG,
+        }
+
+        local REMOVED_MODULES = {
+            ['ipympl/0.9.3-foss-2023a'] = 'This module has been replaced by ipympl/0.9.3-gfbf-2023a',
+        }
+
+        local masterTbl = masterTbl()
+        local error_msg = ""
+        -- The CUDA messages should only be shown if the accelerator stack is NOT being used
+        if not using_eessi_accel_stack() then
+            for _, module in pairs(masterTbl.pargs) do
+                if RELOCATED_CUDA_MODULES[module] ~= nil then
+                    error_msg = error_msg .. module .. ': ' .. RELOCATED_CUDA_MODULES[module] .. '\\n\\n'
+                end
+            end
+        end
+        for _, module in pairs(masterTbl.pargs) do
+            if REMOVED_MODULES[module] ~= nil then
+                error_msg = error_msg .. module .. ': ' .. REMOVED_MODULES[module] .. '\\n\\n'
+            end
+        end
+        if error_msg ~= "" then
+            LmodError('\\n' .. error_msg .. 'If you know what you are doing and you want to ignore this check for removed/relocated modules,, set $EESSI_SKIP_REMOVED_MODULES_CHECK to any value.')
```

Comment:
typo: double `,,` at the end of this line?

Reply:
I can rebuild and re-deploy since these files are always overwritten